core/src/main/kotlin/com/malinskiy/marathon/analytics/tracker/remote/influx/InfluxDbTracker.kt

```diff
@@ -19,7 +19,7 @@ internal class InfluxDbTracker(private val influxDb: InfluxDB) : NoOpTracker() {
                 .tag("package", testResult.test.pkg)
                 .tag("class", testResult.test.clazz)
                 .tag("method", testResult.test.method)
-                .tag("deviceSerial", device.serialNumber)
+                .tag("deviceSerial", device.deviceLabel)
```

Comment:
Is there a reason why we don't add this as an additional tag instead of masking serial?